### PR TITLE
Update groups.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Added description for Director-level expectations

### CODE REVIEW

File: operations/workplace/groups.md [hunk 1]
```diff
@@ -11,3 +11,21 @@ This page describes different leadership groups at Mattermost.
...
+* **Exceptionally strong recruiter, manager, coach upleveling their team and other teams** - Can source, hire, manage and continually uplevel teams and functions. Sought as a resource/mentor cross-organizationally to help in recruiting, vetting, coaching, managing. 
+ * **Source, hires, develops, manages, up-levels individuals and teams** - Experienced manager to hire and develop the best. Continually improves team and individual performance month-by-month. Recognized cross-organizationally as a strong leader and coach that people seek for advice. 
+* **Self-aware and builds teams as sum of strengths** - Strong sense of strengths and opportunities for self and team, and organizes and recruits to create organizations that are the sum of their strengths. Vocally self-aware. 
+* **Owns vital functions of the business** - Engineering (headcount, scope, leadership, typically manager of managers), marketing (budget, strategy, scale of execution), sales (territory, material quotas, headcount), operations (vital systems, expertise, liability, budget, headcount)
```
Comment (amynicol1985) [inline operations/workplace/groups.md]: @natalie-hub - Line 22 and Line 29 are repeats. Not sure I agree with the way the information is laid out.

Comment (lynn915) [reply operations/workplace/groups.md]: Only comment is the repeated content in 22 and 29.

**Review by lynn915 — DISMISSED** [review]

File: operations/workplace/groups.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+* Consists of MLT plus heads of key functions selected by MLT members to represent team.
+* Included in [MLT Extended](https://community.mattermost.com/private-core/channels/mlt--extended) channel.
+
+## Leadership roles
+
+Members of any management group are expected to embody and model leadership values and our [remote principles](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost#we-run-as-a-distributed-high-performance-team-using-a-few-principles).
+
+### Director group
+
+**What's new at Director level?**
```
Comment (justinegeffen) [inline operations/workplace/groups.md]: I'd remove this as there's no preceding context.

**Comment by justinegeffen** [comment]
@amynicol1985, @lynn915 are we good to go on this?

**Comment by amynicol1985** [comment]
@justinegeffen - good to go from my side

**Comment by lynn915** [comment]
@justinegeffen good to go on my end too.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**